tests/test_evaluate.py

```diff
@@ -185,3 +185,13 @@ def test_metric_y_types(
     test_single_task_benchmark_clf.metrics = [Metric.f1]
     result = test_single_task_benchmark_clf.evaluate(y_pred=predictions, y_prob=probabilities)
     assert result.results.Score.values[0] == Metric.f1.fn(y_true=test_y, y_pred=predictions)
+
+
+def test_metrics_docking(test_docking_benchmark: SingleTaskBenchmarkSpecification, caffeine, ibuprofen):
+    _, test = test_docking_benchmark.get_train_test_split()
+
+    predictions = np.array([caffeine, ibuprofen])
+
+    result = test_docking_benchmark.evaluate(y_pred=predictions)
+    for metric in test_docking_benchmark.metrics:
+        assert metric in result.results.Metric.tolist()
```

Comment:
**Suggestion:** Add sanity checks for extremes.

For example, with: 
- `predictions = np.array([caffeine, ibuprofen])`, coverage should be 1.0. 
- `predictions = np.array([ibuprofen, caffeine])`, coverage should be 0.0
  - or should this actually fail? Not the same number of atoms after all?